src/Components/RightSidebar.elm

```diff
@@ -45,3 +46,21 @@ view rightSidebarExtended rightSidebarMsg panelHeight connection =
             [ height (px panelHeight), width (px 50), Background.color Colors.playlistBackground, alignRight, htmlAttribute <| Html.Attributes.style "pointer-events" "all" ]
             [ Input.button [ centerX, height (px 50) ] { onPress = Just rightSidebarMsg, label = Element.text "<" }
             ]
+
+rightSidebarMenuDropDown : Bool -> List (Attribute msg)
+rightSidebarMenuDropDown rightSidebarMenu =
```

Comment:
I think it would be good to name `rightSidebarMenu` something along the lines of `isRightSidebarMenuVisible` or `showRightSidebarMenu`. As it was very unclear to me, what this variable was about, when I saw it in the other places that the name is used.